clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -47,7 +46,7 @@
      * @return A new NodeApiVersions object.
      */
     public static NodeApiVersions create() {
-        return create(Collections.EMPTY_LIST);
+        return create(Collections.<ApiVersion>emptyList());
```

Comment:
Oh, interesting.  I guess this is considered an unchecked conversion, so we should be using the emptyList() form.  Thanks for finding this.